### ISSUE TITLE
Add a general easy option to preserve page breaks (or custom command to easily preserve them at important places)

### ISSUE THREAD

**Issue by claell** [michal-h21/tex4ebook#117]
Currently, tex4ebook doesn't try to preserve page breaks (also see https://tex.stackexchange.com/questions/311339/tex4ebook-and-page-breaks).

They can however be very good for formatting, also for EPUBs. I assume that is also why there seems to be page breaks by default around the TOC or between chapters.

Thus, for other similar uses (titlepage, ...), I suggest to add an easy option for users to preserve the page breaks.

There are likely several ways, but I think a better way than the status quo will be beneficial to many users.

**Comment by michal-h21** [comment]
The problem is that this is not a good idea, we want to ignore `\newpage` or `\clearpage` by default, as these are  used internally a lot, and it could produce spurious elements, or even invalid markup. So it is best to custom commands and CSS, as in the linked answer.

**Comment by claell** [comment]
Okay, I understand. The issue still seems to remain relevant.

Either some option to set to preserve page breaks at certain places (or a custom command) would be probably better options.

Ideally, `tex4ebook` could provide such an option or command. That way, users don't need to copy code from Stack Exchange (which increases complexity and maintenance effort on their side). This would also ensure that the best option is used (since there might be better and worse options to achieve similar results).

**Comment by michal-h21** [comment]
OK, I've added the `\epubpage`  command. It will produce `\cleapage` in the PDF mode, and page break in Epub.

**Comment by claell** [comment]
Great, thank you! Then, I'll close this.

**Comment by claell** [comment]
When testing, I noticed that I also use the `titlepage` environment, which probably has a `\clearpage` inserted at the end. I don't want to modify that environment. As the workaround with custom configuration that redefines the `clearpage` command might be causing unexpected errors, I solved it for now by just adding another `\epubpage` command after the environment.

In usage, this will probably result in two `\clearpage` commands being inserted in PDF mode (and in `tex4ebook`, the first `\clearpage` from the environment is ignored and the `epubpage` inserts a page break). As two `clearpage` commands following each other seem to be "fine", I'll use that solution for now. Seems to be the most robust way to deal with this.
